models/transfers/optimism/erc20/transfers_optimism_erc20_rolling_hour.sql

```diff
@@ -0,0 +1,17 @@
+{{ config(
+        tags = ['dunesql'],
+        alias = alias('rolling_hour'))
+}}
+
+        SELECT
```

Comment:
I might be reading this wrong, but wouldn't we need this to be on an incremental run as well? Otherwise it will only run when we deploy it?

Reply:
it defaults to a view if we don't specify in config block, so the data will be live and just refreshed at runtime on the app

Reply:
Ah ok that was a derp from me, I was thinking for a sec it would run once create the table and leave it in a stale state. Makes sense!